src/common/components/Navbar/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Children, cloneElement, isValidElement } from 'react';
+import PropTypes from 'prop-types';
+
+import ResponsiveNavigation from '../../../packages/responsive-navigation';
+
+import { Icon, Popover, Menu } from '../';
+
+const Navbar = ({ children, trigger }) => (
+  <ResponsiveNavigation>
+    {({ activePath, hideNavigation, isMobile, isNavigationVisible, showNavigation }) => {
+      if (children.type !== Navbar.Menu || !isValidElement(Children.only(children))) {
```

Comment:
Is there a value for which Children.only can pass and the result can be an invalid element?

Reply:
thats a great question, React docs is not helpful here. Will dig in the React's source code

Reply:
Ok so actually `Children.only` uses the `isValidElement` https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/packages/react/src/ReactChildren.js#L387